scene/gui/color_mode.h

```diff
@@ -61,6 +63,7 @@ class ColorModeHSV : public ColorMode {
 public:
 	String labels[3] = { "H", "S", "V" };
 	float slider_max[4] = { 359, 100, 100, 255 };
+	float cached[4];
```

Comment:
It should have explicit names like it had in ColorPicker. idk why you used an array.

Reply:
fixed!